plugin/src/main/java/org/opensearch/ml/utils/RestActionUtils.java

```diff
@@ -308,4 +316,27 @@ public static void wrapListenerToHandleSearchIndexNotFound(Exception e, ActionLi
         }
     }
 
+    public static boolean isMultiTenant(ClusterService clusterService, Settings settings) {
```

Comment:
minor: `isMultiTenant` -> `isMultiTenantEnabled`

Move to MLFeatureEnabledSetting ?

Reply:
Sure, will address in the next follow up PR.